assets/app/styles/_core.less

```diff
@@ -203,6 +203,19 @@ html, body {
   }
 }
 
+.nav-tabs {
+  font-size: @font-size-base;
+  a {
+    // ui.bootstrap.tabs doesn't set the href attr, so the cursor style is
```

Comment:
Hash URIs cause the router to reload, but it works well with query parameters if you set `reloadOnSearch: false`.